Confus/ConfusTest/FlagTest.cpp

```diff
@@ -0,0 +1,71 @@
+#include "stdafx.h"
+
+#include <string>
+#include "../Confus/Flag.h"
+#include "../ConfusShared/Physics/PhysicsWorld.h"
+
+#include "CppUnitTest.h"
+
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+
+namespace Microsoft 
+{
+    namespace VisualStudio 
+    {
+        namespace CppUnitTestFramework 
+        {
+            template<>
+            static std::wstring ToString<irr::core::vector3df>(const irr::core::vector3df  & vector) 
+            {
+                return std::to_wstring(vector.X) + L" " + std::to_wstring(vector.Y) + L" " + std::to_wstring(vector.Z);
+            }
+        }
+    }
+}
+
+namespace ConfusTest
+{
+    TEST_CLASS(FlagTest)
+    {
+        Confus::Physics::PhysicsWorld Physics;
+        Confus::Flag* Flag;
+        irr::IrrlichtDevice* Device;
+
+    public:
+        FlagTest() : Physics(Device)
```

Comment:
That's probably not going to work. The pointer to the IrrlichtDevice is set in TEST_METHOD_INITIALIZE, but the initializer list is ran before that. The physics world will never have the up to date irrlicht device. Fine for now anyway, since its only used for debug drawing